staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/connection_test.go

```diff
@@ -31,7 +31,6 @@ import (
 	"testing"
 
 	"k8s.io/legacy-cloud-providers/vsphere/vclib"
-	"k8s.io/legacy-cloud-providers/vsphere/vclib/fixtures"
```

Comment:
@andrewsykim @cheftako re: this commit

Inspecting the runtime caller to try to locate the current package is a really brittle approach and will be broken in the future when we wish to compile with `-trimpath` anyhow, let alone attempts to reference bazel specific logic.

The canonical go approach is to just use relative paths and a `testdata` directory, go will set the current working directory to the package under test.

These files are only used in two packages in the same heirarchy, so this works trivially.